### PR TITLE
ZING-9177: Add new Event messages from zing-proto

### PR DESCRIPTION
**Pull request by pcarinhas**
* We added new messages to zing-proto's event.proto fields which
  will match the additions to streaming data_receiver.proto fields.
* We don't modify any of the older message handling.
* This will support zing-proto 11.1.2 version

### CODE REVIEW

File: core/src/main/java/org/zenoss/zep/zing/ZingEvent.java [hunk 1]
```diff
@@ -182,13 +189,56 @@ public Event toZingEventProto() {
...
-            //-----------
-            //  Metadata
-            //-----------
-            if (!ZingUtils.isNullOrEmpty(this.severity))
-                b.putMetadata( ZingConstants.SEVERITY_KEY, ZingUtils.getScalarArray(this.severity));
-            if (!ZingUtils.isNullOrEmpty(this.status))
+            //--------------------------
+            //  Status Map
+            //--------------------------
+            Map<String, Status> statusMap = new HashMap<String, Status>();
```
Comment (sockterrier) [inline core/src/main/java/org/zenoss/zep/zing/ZingEvent.java:195]: make statusMap and severityMap into static members of ZingEvent and populate them with a static initializer.

File: core/src/main/java/org/zenoss/zep/zing/ZingEvent.java [hunk 2]
```diff
@@ -25,6 +29,27 @@
 
     private static final Logger logger = LoggerFactory.getLogger(ZingEvent.class);
 
+    private static final BoolValue boolValueTrue = BoolValue.newBuilder().setValue(true).build();
+    private static final BoolValue boolValueFalse = BoolValue.newBuilder().setValue(false).build();
+    private static final Map<String, Status> statusMap = new HashMap<String, Status>();
+    private static final Map<String, Severity> severityMap = new HashMap<String, Severity>();
+    static {
+        statusMap.put("NEW", Status.STATUS_OPEN);                  // Also: Ensure Events.setAcknownledged("False") ?
+        statusMap.put("ACKNOWLEDGED", Status.STATUS_OPEN);         // Also: Ensure Events.setAcknownledged("True") ?
```
Comment (sockterrier) [inline core/src/main/java/org/zenoss/zep/zing/ZingEvent.java:38]: shoot, i missed this the first time. fix spelling to "setAcknowledged"

**Comment by pcarinhas** [comment]
please retest
please recheck

**Comment by pcarinhas** [comment]
retest this

**Comment by pcarinhas** [comment]
retest this

**Comment by pcarinhas** [comment]
retest this

**Comment by pcarinhas** [comment]
retest this

**Comment by pcarinhas** [comment]
jenkins retest this

**Comment by pcarinhas** [comment]
retest this

**Comment by pcarinhas** [comment]
retest this

**Pull request merged**